charts/message-parser/templates/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-message-parser-app
+  labels:
+    app: message-parser-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: message-parser-app
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: message-parser-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/message-parser:{{ default "latest" .Values.image.tag }}
```

Comment:
I think you can set the default dynamically, with

`{{ .Values.image.tag | "latest" }}`

I'm not sure the template will be able to parse it as written.

Reply:
It did when I ran it, but it looked really weird...

Reply:
changed